### ISSUE TITLE
Cannot read property 'Parser' of undefined

### ISSUE THREAD

**Issue by schoettler** [cucumber/cucumber-electron#6]
Running the node example on [cucumber-js ](https://github.com/cucumber/cucumber-js/blob/master/docs/nodejs_example.md)

Gives me the following error:
```
/path/to/project/node_modules/cucumber-electron/renderer/patches/gherkin.js:6 Uncaught TypeError: Cannot read property 'Parser' of undefined
```
When running:

`./node_modules/.bin/cucumber-electron test/path/to/file.feature --electron-debug`

**Comment by aslakhellesoy** [comment]
It could be that the latest gherkin 4.1.1 release is the culprit. Haven't verified. Can you try pinning to 4.0.0 for now?

**Comment by schoettler** [comment]
Thanks, @aslakhellesoy that fixed the issue!

**Comment by joshski** [comment]
Thanks both, I've released cucumber-electron 2.1.4 which does the same for now.
